launch/launch/actions/execute_local.py

```diff
@@ -90,7 +90,7 @@ def __init__(
         sigkill_timeout: SomeSubstitutionsType = LaunchConfiguration(
             'sigkill_timeout', default=5),
         emulate_tty: bool = False,
-        output: Text = 'log',
+        output: Union[Text,SomeSubstitutionsType] = 'log',
```

Comment:
@Khush-dev `SomeSubstitutionsType` ais a type `Union` already including `Text`: 
```suggestion
        output: SomeSubstitutionsType = 'log',
```